build.gradle.kts

```diff
@@ -73,7 +73,7 @@ dependencies {
     jarLibImplementation("dev.fastmc:fastmc-common:1.1-SNAPSHOT:java8")
     jarLibImplementation("dev.fastmc:mem-util:1.0-SNAPSHOT")
 
-    implementation("com.github.cabaletta:baritone:1.2.14")
+    implementation("cabaletta:baritone-deobf-unoptimized-mcp-dev:1.2")
     jarLib("cabaletta:baritone-api:1.2")
```

Comment:
The jar lib must also be updated to the matching version.

Reply:
The jar lib was already up to date with the latest version of baritone. It was just the library that was out of date. To confirm, using the masterbranch without changes, I used the following commands in game which are present in the jarlibs, but not in the current outdated baritone library. Let me know if any other changes are needed :)
- layerHeight
- startAtLayer
- skipFailedLayers
- buildOnlySelection
- buildIgnoreDirection
- assumeExternalAutoTool
- autoTool